Sources/EpoxyBars/BarModel/AnyBarModel.swift

```diff
@@ -39,6 +39,10 @@ extension AnyBarModel: BarModeling {
 
 extension AnyBarModel: DataIDProviding {}
 
+// MARK: DidSelectProviding
+
+extension AnyBarModel: DidSelectProviding {}
+
```

Comment:
I think we can leave this out for now—I don't believe it will have any effect as-is
```suggestion
```